### ISSUE TITLE
Add withoutCache?

### ISSUE THREAD

**Issue by canadaduane** [lukewestby/elm-http-builder#2]
Apparently this works as a means to prevent all browsers from caching GET requests (see http://stackoverflow.com/questions/367786/prevent-browser-caching-of-jquery-ajax-call-result#comment47453110_367796):

```
withHeader "Cache-Control" "max-age=0,no-cache,no-store,post-check=0,pre-check=0"
```

Maybe make that in to a `withoutCache` shortcut?

Alternatively, add a cache-busting `_={timestamp}` to the end of the URL. See http://stackoverflow.com/a/367827/159344.

**Comment by lukewestby** [comment]
I was thinking of adding the timestamp query param since that involves Tasks and would muddle up people's code if they did it themselves. Since `withHeader` already exists I think folks would be okay adding a Cache-Control header manually. What do you think?

**Comment by lukewestby** [comment]
Just published 1.1.0 with this function!
